discovery-provider/plugins/notifications/src/processNotifications/mappers/addTrackToPlaylist.ts

```diff
@@ -55,13 +66,16 @@ export class AddTrackToPlaylist extends BaseNotification<AddTrackToPlaylistNotif
     const playlistRes: Array<{
       playlist_id: number
       playlist_name: string
-      playlist_owner_id: number
+      playlist_owner_id: number,
+      playlist_contents: PlaylistContents,
     }> = await this.dnDB
-      .select('playlist_id', 'playlist_name', 'playlist_owner_id')
+      .select('playlist_id', 'playlist_name', 'playlist_owner_id', 'playlist_contents')
       .from<PlaylistRow>('playlists')
-      .where('is_current', true)
+      .whereIn("is_current", ["true", "false"])
```

Comment:
no need for is_current check

Reply:
this is for the db index, did the same thing on verified uploads plugin

Reply:
oh yea i think in this case it doesn't matter cuz it's ordering by block, the query plan is the same

Reply:
sure i can remove it for now